apps/service-catalog/app/loading.tsx

```diff
@@ -0,0 +1,22 @@
+import { CenterContainer, PageBanner } from '@catalog-frontend/ui';
+import { localization } from '@catalog-frontend/utils';
+import { Heading, Spinner } from '@digdir/design-system-react';
+
+export default function Loading() {
```

Comment:
```suggestion
const Loading = () => {
```

Reply:
Skal vi ha en egen page til dette?

Reply:
https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming

Nextjs har egen loading ui.